roles/pulibrary.dspace/defaults/main.yml

```diff
@@ -1,12 +1,14 @@
 ---
+deploy_user: "dspace"
 dspace_version: 6.3
 tomcat8_enabled: true
 tomcat8_hostname: localhost
 tomcat8_server_port: 8005
 tomcat8_catalina_port: 8983
+tomcat_http_port: 8080
+tomcat_ajp_port: 8009
 tomcat8_catalina_redirect_port: 8443
 tomcat_shutdown_port: "{{ tomcat8_server_port }}"
-tomcat_port: 8080
-tomcat_connection_timeout: 20000
-tomcat_redirect_port: "{{ tomcat8_catalina_redirect_port }}"
-tomcat_ajp_port: 8009
+tomcat_service_name: "apache-tomcat"
+tomcat_version: 7.0.104
```

Comment:
I assume that this is the version set by Atmire, but I would be curious as to why we cannot use Tomcat 8.x releases.

Reply:
Atmire can answer this.